reinforcement_learning/rlclientlib/live_model.cc

```diff
@@ -26,6 +26,11 @@ namespace reinforcement_learning
     new live_model_impl(config, fn, err_context, trace_factory, t_factory, m_factory, sender_factory));
   }
 
+  live_model::live_model(live_model&& model) {
+    _pimpl = std::move(model._pimpl);
```

Comment:
nit:  destructor is trivial now but that might change.  would a swap be better for these variables?